core/src/main/java/org/mskcc/cbio/oncokb/util/EvidenceUtils.java

```diff
@@ -988,7 +1012,7 @@ public static List<EvidenceQueryRes> processRequest(List<Query> requestQueries,
                         );
                     }
                 }
-                query.setEvidences(new ArrayList<>(StringUtils.isEmpty(query.getQuery().getTumorType()) ? updatedEvidences : keepHighestLevelForSameTreatments(updatedEvidences, requestQuery.getReferenceGenome(), query.getExactMatchedAlteration())));
+                query.setEvidences(new ArrayList<>(StringUtils.isEmpty(query.getQuery().getTumorType()) ? updatedEvidences : keepHighestLevelForSameTreatments(updatedEvidences, requestQuery.getReferenceGenome(), query.getExactMatchedAlteration(), query.getQuery().getTumorType())));
```

Comment:
Would using `matchedTumorType` work too? Can save a call to the database in EvidenceUtils L717